openmdao/test/util.py

```diff
@@ -154,4 +158,30 @@ def assert_no_force_fd(group):
     subs = [s.pathname for s in group.subsystems(recurse=True, include_self=True)
             if s.fd_options['force_fd']]
     assert not subs, "One or more systems are using " \
-                     "fd_options['force_fd']=True: " + str(subs)
\ No newline at end of file
+                     "fd_options['force_fd']=True: " + str(subs)
+
+
+class ConcurrentTestCaseMixin(object):
```

Comment:
just curious why you did this as a MixIn instead of just adding it to MPITestCase